cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -36,11 +36,11 @@ var (
 )
 
 func init() {
-	utilruntime.Must(componentconfig.AddToScheme(configScheme))
-	utilruntime.Must(componentconfigv1alpha1.AddToScheme(configScheme))
+	utilruntime.Must(kubeschedulerconfig.AddToScheme(configScheme))
```

Comment:
You can/should create a `pkg/scheuler/apis/config/scheme` package with global variables Scheme and Codecs instead and use them.

Reply:
like how kubeadm does it